packages/block-library/src/social-links/edit.js

```diff
@@ -100,6 +100,7 @@ export function SocialLinksEdit( props ) {
 	const innerBlocksProps = useInnerBlocksProps( blockProps, {
 		allowedBlocks: ALLOWED_BLOCKS,
 		orientation: 'horizontal',
+		__experimentalCaptureToolbars: true,
```

Comment:
@mtias, I have an idea for controls injected from the framework through `supports` in `block.json` like `align`. We could put also `__experimentalCaptureToolbars` in `supports` object to act as a way to decide if those general controls should also be exposed to children. An alternative would be to include `__experimentalExposeToChildren` setting in `supports` object.

Reply:
> __experimentalCaptureToolbars

I'm slightly confused by usage of this. "Capture toolbars" is for the UI space occupied by the _parent_ toolbar to display the toolbars of any child. This is great when having deeply nested navigation blocks where having the toolbar show up on the deeply nested child gets very clunky.

If we're implementing a feature which is essentially the inverse (ie: _child_ toolbar UI space includes controls form the _parent_) then it should probably have a different name/setting. `__experimentalExposeToChildren` seems good.

Reply:
My current thinking is that for "capture toolbars" to make sense, it also needs to expose tools to the children, otherwise it feels a bit awkward to use.

Reply:
Yes, this part is something that was only added yesterday so I'm open seek a better API here even under the experimental flag.

Let me explain my reasoning better. My assumption was that you want to combine the toolbar capturing with exposing children to have more space to display controls. Given that they both work a bit differently at the moment, I'm fine following your recommendation:
- `__experimentaExposeControlsToChildren` - at the moment it is limited only to children, in effect, it doesn't apply to deeper descendants 

Aside, I like the idea that `useInnerBlocksProps` would be able to read some settings from the `supports` object so maybe having `__experimentalCaptureToolbars` also there isn't that bad idea. I saw many requests where people wanted to control allowed blocks or template lock settings.

Reply:
Let me know what do you think about https://github.com/WordPress/gutenberg/pull/33955#issuecomment-899355526.